app/views/categories/index.html.slim

```diff
@@ -1,19 +1,11 @@
 .row
-  .col-md-10
+  .col-md-7
     h3.header.text-center= t('.all_categories')
-  .col-md-2
+  .col-md-5.d-flex
     = link_to t('.new_category'), new_category_path, { remote: true, 'data-toggle' => "modal", 'data-target' => '#modal-window', class: 'btn btn-info pull-right' }
```

Comment:
@sinpai use new ruby syntax instead of rockets

Reply:
Already changed according to your previous comment in PR before.